### DISCUSSION TITLE
Time it took to render missing... adetailer settings not being transfered wen open in Txt2Txt

### DISCUSSION THREAD

**Discussion by Akossimon** [zanllp/infinite-image-browsing#734] [Ideas]
i would have 2 questions:

1) i am always balancing how long an image took to render, but that informations seesm to be missing in this extension when viewing an image, is this a hidden feature that needs ti be activated?

2) when i choose an image from this browser and choose the option to open it in txt2img , it does do the right thing with main prompt and negative in that txt2img UI, but it ignores other settings , for example adetailer settings and its parameters , or prior xyz plot settings that were used to create the image in the first place. am i doing something wrong?

**Comment by zanllp** [comment]
There are no hidden features that need to be enabled. IIB actually just forwards without modifying anything.

**Reply by akos111** [reply]
if such possibly hidden options do not exist as you say, then it ignores such adetailer information alltogether.... and yet sometimes some adetailer tabs parameters settings do transfer over from adetailer, when one chooses to open the selected image prompt in the txt2img UI again to run an image again,.....but sometimes they do not.Also in the TASK QUEY there is no column for adetailer info neither, which would be a huge advantage to see such adetailer column there...
